modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: Procedure
+[id="deploying-book-info"]
+= Deploying the Bookinfo application
+:context: ossm-pdeploying-book-info
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+* You have access to the {ocp-short-name} CLI (oc).
+* You have installed the {SMProductName} Operator, and deployed {istio}.
+* You have created the {istio} and IstioCNI resources.
```

Comment:
If I understand correctly, {istio} is deployed only after creating the {istio} resource. So should we rephrase the sentence in Line 11 which says that {istio} is deployed.

Reply:
Thanks @rh-tokeefe. Shall we modify the sentence in Line 12 as follows.

```
You have created the IstioCNI resource and the operator has deployed the necessary IstioCNI pods.
```